app/components/mandatees/mandatees-panel/mandatees-panel-view.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+
+export default class MandateesMandateesPanelEditComponent extends Component {
+  @service currentSession;
+
+  get isAdminOrSecretarie() {
+    return this.currentSession.isAdmin || this.currentSession.isSecretarie;
```

Comment:
Should the `Kanselarij` users also have this? Perhaps even `OVRB` (I think they also see mandatees and might be interested in knowing which one is selected)?

Also it might be easier to add this as a permission (a string in `app/config/permissions.js`, e.g. `view-mandatees-with-range`) added for all the profiles and then here you would do `this.currentSession.may('view-mandatees-with-range')`.

Reply:
@tdn Perhaps you know about other profiles that need this.

Other than that, the PR looks great!